.github/workflows/maven-pr-review.yml

```diff
@@ -0,0 +1,17 @@
+name: Pull Request Approved
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.mergeable == true && github.event.review.state == 'approved'
+    steps:
+      - name: Warn about missing labels
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'release') && !contains(github.event.pull_request.labels.*.name, 'documentation') }}
+        uses: thollander/actions-comment-pull-request@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          message: "This pull request hasn't been labeled as `release` nor `documentation`. Please ensure this is intentional before merging"
```

Comment:
```suggestion
          message: "This pull request hasn't been labeled as `release` nor `documentation`. Please ensure this is intentional before merging."
```

Reply:
In comment sections and such, it's usual to not end messages in a period to make them sound friendlier. Since other messages in our repos already did this (in the version specification warning, as well as the greeting message), I decided to do the same here.

It would work either way, I don't know which would be more appropiate since it's a bot, but it would be good to have a consistent style across messages

Reply:
The correct way to do this is to include the period in all sentences. Are we sure to want a friendlier tone in a Pull Request?